Source/Scene/PointCloud3DTileContent.js

```diff
@@ -781,8 +785,11 @@ define([
         var hasNormals = content._hasNormals;
         var hasBatchIds = content._hasBatchIds;
         var backFaceCulling = content._backFaceCulling;
+        var normalShading = content._normalShading;
         var vertexArray = content._drawCommand.vertexArray;
 
+        var normalsEnabled = hasNormals && normalShading;
```

Comment:
Are you referring to the part around line 1022 (https://github.com/AnimatedRNG/cesium/blob/eb7b76ece10dd4422a9377b94df8ac725c97b730/Source/Scene/PointCloud3DTileContent.js#L1022)?